cmd/image/import/import.go

```diff
@@ -1,16 +1,93 @@
 package _import
 
 import (
-	"fmt"
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client"
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
 	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"os"
+	"strings"
 )
 
 var Cmd = &cobra.Command{
 	Use:   "import",
 	Short: "Import the image into PowerVS instances",
-	Long:  `Import the image into PowerVS instances`,
+	Long: `Import the image into PowerVS instances
+pvsadm image import --help for information
+
+# Set the API key or feed the --api-key commandline argument
+export IBMCLOUD_API_KEY=<IBM_CLOUD_API_KEY>
+
+Examples:
+
+# importing image from default region and using default storage type
+pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --accesskey <ACCESSKEY> --secretkey <SECRETKEY> --s3-image-name rhel-83-10032020.ova.gz --image-name test-image
```

Comment:
Should the option `s3-image-name` be object-name or `image-name` and instead `image-name` be `imported-image-name`

Reply:
I agree with @bpradipt we can rename to `object-name` and somehow feel we can just have a `--name` because we already have the command with the name `image import`